version.php

```diff
@@ -27,7 +27,7 @@
 /** @var stdClass $plugin */
 $plugin->component = 'filter_ally';
 $plugin->release   = '4.4.6';
-$plugin->version   = 2025040400;
+$plugin->version   = 2025040401;
```

Comment:
Note - the version is only incremented by the most minor point since this is a backport fix.
We can't give it the latest version because that would prevent upgrades from being triggered correctly if someone installed the plugin from a higher branch version.